webapp/tests/mocha/unit/bootstrapper.spec.js

```diff
@@ -72,6 +72,11 @@ describe('bootstrapper', () => {
       show: sinon.stub()
     });
 
+    if (typeof fetch !== 'undefined') {
```

Comment:
Instead of altering the global fetch, can you use `rewire.__set__('fetch')` within the scope of the bootstrapper only?

Reply:
Done.